docs/section5/Rule5-39.md

```diff
@@ -0,0 +1,38 @@
+
+# Envelope - Rule 5-39  
+
+**Rule ID:** 5-39  
+**Rule Description:** Automatically controlled dynamic glazing may be modeled. Manually controlled dynamic glazing shall use the average of the minimum and maximum SHGC and VT.  
+**Rule Assertion:** P-RMR subsurface: dynamic_glazing_type = expected value  
+**Appendix G Section:** Section G3.1-5(a)5 Building Envelope Modeling Requirements for the Proposed design  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** None  
+
+**Manual Check:** Yes  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:** None
+
+## Rule Logic:  
+
```

Comment:
include some guidance on what to do for applicability check.

Reply:
Updated.